### PR TITLE
[ID] Update `Bot account`

### PR DESCRIPTION
**Pull request by ZeroPyrozen**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Bot_account/id.md [hunk 1]
```diff
@@ -11,7 +9,7 @@ tags:
 
 # Akun bot
 
-**Akun bot** adalah jenis akun osu! yang dijalankan oleh layanan otomatis dan bukan orang sungguhan. Jika diperiksa lewat situs web, akun-akun ini memiliki lencana berlabel putih dan bertuliskan `BOT` dan hanya bagian `me!` yang terlihat di profil mereka. Akun bot paling sering digunakan untuk meng-host "bot obrolan" yang menawarkan berbagai fungsi kepada komunitas melalui *channel* ruangan [multiplayer](/wiki/Gameplay/Multiplayer) atau pesan pribadi.
+**Akun bot** merupakan sebuah akun osu! khusus yang dijalankan oleh layanan otomatis dan bukan orang sungguhan. Jika dilihat di situs web, akun-akun ini memiliki lencana berlabel putih dan bertuliskan `BOT` dan hanya bagian `me!` yang terlihat di profil mereka. Akun bot paling sering digunakan untuk meng-*host* "bot obrolan" yang menawarkan berbagai fungsi kepada komunitas melalui *channel* ruangan [multiplayer](/wiki/Gameplay/Multiplayer) atau pesan pribadi.
```
Comment (hikikuman) [inline wiki/Bot_account/id.md:12]:
```suggestion
**Akun bot** merupakan sebuah akun osu! khusus yang dijalankan secara otomatis, bukan dijalankan oleh orang sungguhan. Jika dilihat di situs web, akun-akun ini memiliki lencana berlabel putih dan bertuliskan `BOT` dan hanya bagian `me!` yang terlihat di profil mereka. Akun bot paling sering digunakan untuk meng-*host* "bot obrolan" yang menawarkan berbagai fungsi kepada komunitas melalui kanal lobi [multiplayer](/wiki/Gameplay/Multiplayer) atau pesan pribadi.
```

File: wiki/Bot_account/id.md [hunk 2]
```diff
@@ -32,20 +30,21 @@ Jika kamu ingin membuat sebuah bot obrolan, kamu dapat memulainya dengan menjala
...
 - Bot digunakan oleh setidaknya 50 pengguna berbeda setiap bulannya
 - Bot mematuhi batasan pengiriman pesan yang dimiliki akun pribadi
 - Bot tidak mengirim pesan apa pun di *channel* publik
-- Bot digunakan demi kepentingan dan keuntungan komunitas
+- Bot tidak mengirim pesan pribadi tanpa seizin pengguna
+- Bot digunakan demi kepentingan dan keuntungan komunitas osu!
 
 ### Mengajukan permintaan
 
 Jika bot kamu memenuhi kriteria yang tertera di atas, kamu dapat melakukan pengajuan pembuatan akun bot untuk bot tersebut.
```
Comment (hikikuman) [inline wiki/Bot_account/id.md:43]:
```suggestion
Jika bot kamu memenuhi kriteria yang tertera di atas, kamu dapat melakukan pengajuan pembuatan akun bot.
```

File: wiki/Bot_account/id.md [hunk 3]
```diff
@@ -32,20 +30,21 @@ Jika kamu ingin membuat sebuah bot obrolan, kamu dapat memulainya dengan menjala
...
 
 Silakan mengirim email ke [accounts@ppy.sh](mailto:accounts@ppy.sh) dengan judul `Bot Account Request`. Email ini harus dikirim dari alamat email yang tersambung dengan akun osu! kamu.
```
Comment (hikikuman) [inline wiki/Bot_account/id.md:45]:
```suggestion
Silakan kirim email ke [accounts@ppy.sh](mailto:accounts@ppy.sh) dengan judul `Bot Account Request`. Email ini harus dikirim dari alamat email yang tersambung dengan akun osu! kamu.
```

**Review by hikikuman — CHANGES_REQUESTED** [review]
`*channel*` diganti jadi `kanal` gimana?

File: wiki/Bot_account/id.md [hunk 4]
```diff
@@ -32,20 +30,21 @@ Jika kamu ingin membuat sebuah bot obrolan, kamu dapat memulainya dengan menjala
 
 Sebelum mengirim pengajuan, tim dukungan mengharuskan bot tersebut memnuhi kriteria berikut:
 
-- Bot telah berjalan di akun kamu selama setidaknya 6 bulan
-- Bot bersifat sepenuhnya *open source* dan memiliki dokumentasi publik lengkap
+- Bot telah berjalan di akunmu selama setidaknya 6 bulan
+- Bot bersifat sepenuhnya *open-source* dan memiliki dokumentasi publik lengkap
 - Bot digunakan oleh setidaknya 50 pengguna berbeda setiap bulannya
 - Bot mematuhi batasan pengiriman pesan yang dimiliki akun pribadi
 - Bot tidak mengirim pesan apa pun di *channel* publik
```
Comment (mousesalt) [inline wiki/Bot_account/id.md:37]:
```suggestion
- Bot tidak mengirim pesan apa pun di kanal publik
```

**Review by mousesalt — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**